File: api/src/test/java/io/github/wulkanowy/timetable/api/TimetableTest.java
package io.github.wulkanowy.timetable.api;

import org.junit.Test;
import org.junit.Assert;

public class TimetableTest {

    @Test
    public void addition() throws Exception {
        Assert.assertEquals(4, new Timetable().addition(2, 2));
    }
}

File: api/src/test/java/io/github/wulkanowy/timetable/api/TimetableTest.kt
package io.github.wulkanowy.timetable.api

import org.junit.Assert
import org.junit.Test

class TimetableTest {

    @Test
    @Throws(Exception::class)
    fun addition() {
        Assert.assertEquals(4, Timetable().addition(2, 2).toLong())
    }
}
